roadie-kubernetes-cluster-access/templates/cluster-role.yaml

```diff
@@ -17,5 +17,11 @@ roleRef:
   kind: ClusterRole
   name: {{ .Values.clusterRole.name | default (printf "%s-cluster-role" .Release.Name) }}
 subjects:
+  {{ if .Values.serviceAccount.enabled }}
+  - kind: ServiceAccount
+    name: {{ .Values.serviceAccount.name }}
+    namespace: {{ .Values.serviceAccount.namespace }}
```

Comment:
Should be .Release.Namespace to match the service account in the broker.yaml file?

Reply:
So I was thinking this is for when they use an existing service account and use the token. Maybe they should always create a new one though?

Reply:
Ah right sorry. I actually misunderstood this whole commit. You are adding a new feature basically. Previously it could support:
- external access via something like eks iam role access or whatever the equivilent is in gke.
- access via the broker

Now you are adding a feature to generate a service account token...

I wonder should we have a flag like mode? e.g.

mode: broker | service-account | other

Or maybe what you have done is fine. I guess its just a little confusing now which settings I should use for which case. But if we document it well enough maybe its just fine.

Reply:
YeaI'm giving example override files to use in each case

Reply:
There is an option to supply the config items by cli if you want to avoid asking the customer to provide files.

```shell
helm install brian-local \
  --set broker.enabled=true \
  roadie/roadie-kubernetes-cluster-access
```